ui/app/components/empty-state.js

```diff
@@ -0,0 +1,6 @@
+import Component from '@ember/component';
```

Comment:
Just wanted to note, you _could_ just call this `empty.js` if you wanted - we don't need the hyphenated name with the new angle bracket components now.